js/loading.js

```diff
@@ -0,0 +1,13 @@
+
+window.setTimeout(finish, 2500);
+
+function finish() {
+    const spinner = document.getElementById('loading');
+    spinner.classList.add('loaded');
+}
+
+window.onpageshow = function (event) {
+    if (event.persisted) {
+        window.location.reload();
+    }
+};
```

Comment:
最終行は空行にすること。JavaScriptでは問題ないが、言語によっては許容されない場合がある。
```suggestion
};

```

Reply:
理由）改行によって行として認識されるから